src/main/java/pages/CreateAnAccountPage.java

```diff
@@ -0,0 +1,190 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+import utils.TextCreator;
+
+public class CreateAnAccountPage extends PageObject {
+    public CreateAnAccountPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(id = "id_gender2")
+    private WebElement mrsGender;
+
+    @FindBy(id = "id_gender1")
+    private WebElement mrGender;
+
+    @FindBy(id = "customer_firstname")
+    private WebElement firstNameField;
+
+    @FindBy(id = "customer_lastname")
+    private WebElement lastNameField;
+
+    @FindBy(id = "email")
+    public WebElement emailField;
+
+    @FindBy(id = "passwd")
+    private WebElement passwordField;
+
+    @FindBy(id = "days")
+    private Select daysSelect;
+
+    @FindBy(id = "days")
+    private WebElement daysWebelement;
+
+    @FindBy(id = "months")
+    private Select monthsSelect;
+
+    @FindBy(id = "years")
+    private Select yearsSelect;
+
+    @FindBy(id = "newsletter")
+    private WebElement newsletterRadioButton;
+
+    @FindBy(id = "optin")
+    private WebElement optionRadioButton;
+
+    @FindBy(id = "firstname")
+    private WebElement firstNameAddressField;
+
+    @FindBy(id = "lastname")
+    private WebElement lastNameAddressField;
+
+    @FindBy(id = "company")
+    private WebElement companyField;
+
+    @FindBy(id = "address1")
+    private WebElement addressFirstLineField;
+
+    @FindBy(id = "address2")
+    private WebElement addressSecondLineField;
+
+    @FindBy(id = "city")
+    private WebElement cityField;
+
+    @FindBy(id = "id_state")
+    private Select stateSelectList;
+
+    @FindBy(id = "postcode")
+    private WebElement postcodeField;
+
+    @FindBy(id = "id_country")
+    private Select countrySelectList;
+
+    @FindBy(id = "other")
+    private WebElement otherTextArea;
+
+    @FindBy(id = "phone")
+    private WebElement homePhoneField;
+
+    @FindBy(id = "phone_mobile")
+    private WebElement phoneMobileField;
```

Comment:
ujednolić to, na górze jest homePhoneField a na dole phoneMobileField, moze zmienić na mobilePhoneField?